backend/accessibility_templates/aa-002.xml

```diff
@@ -0,0 +1,7 @@
+<AccessibilityAttributes>
+ <SigningAttributes>
+  <Coding href="urn:dvb:metadata:cs:VideoCodecCS:2022:5.1.10"/>
+  <SignLanguage>deu</SignLanguage>
```

Comment:
It would be better to use the code for German Sign Language `gsg`

Reply:
OK, I make this change in an upcoming update.